### ISSUE TITLE
[Suggestion] Reduced spam

### ISSUE THREAD

**Issue by SebSemos** [A11v1r15/Alivent_messager#19]
Hi, Having set all mob death, A lot of them are dumb :p 
I can see few solution: 
A blacklist, with a death reason for a mob. But you'll losse all the infos.
or, like some chat mods do, display the death once and add a little counter to it (like on the picture where I spammed "t" 4 times)
Sadly I haven't found one of these mods working with your mod.

![huge_2024-06-21_01 33 50](https://github.com/A11v1r15/Alivent_messenger/assets/89230340/90c7653b-ced0-4e76-a7ff-4e167f3d28ee)

**Comment by A11v1r15** [comment]
Can you say the chat mods that do this text compression so I can take a look?

**Comment by SebSemos** [comment]
Yep, there is [Chat Patches](https://modrinth.com/mod/chatpatches) (but turned out the compression thing doesn't work at all for me in 1.20.1, I haven't tested in .4 or else that seems to have got an update for this matter.

[Compat chat](https://modrinth.com/mod/compact-chat)  (the one from the picture)  also didin't work for you mod's messages.
But after I opened this issue I came around [Chatutils](https://modrinth.com/mod/chatutils) wich does compact the death messages :)

Still think it could be a cool feature directly from your mod :) 
Main benefit beiing that you don't get randomly a big list of stupid squids covering half the screen :p

**Comment by A11v1r15** [comment]
Oh, so it does compact. I thought I wouldn't "stack" for the same reason as beehives, since while the text is the same, they have that hover event with the entity UUID.
I'm now making a mod for Modfest, so I'll only look at it later
